### PR TITLE
refactor: Apply Wikipedia's style of bolding to multiple pages

### PR DESCRIPTION
**Pull request by redoomed1**
Changes proposed in this PR:

- Apply Wikipedia's style of bolding (recommended [here](https://github.com/privacyguides/privacyguides.org/pull/2620#discussion_r1648632857)) to tool recommendation pages where appropriate
- Reword the description for the Desktop Browsers page so that it's more consistent with [the one for the Mobile Browsers page](https://github.com/privacyguides/privacyguides.org/blob/main/docs/mobile-browsers.md?plain=1#L5)
- Make minor wording changes otherwise not noted above

(Side note: I chose to label this PR as a "refactor" because I'm not sure whether the changes here count as a "feature" for the purposes of the commit message. They're not really fixes or updates either...)

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: docs/email-clients.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ icon: material/email-open
 description: These email clients are privacy-respecting and support OpenPGP email encryption.
 cover: email-clients.webp
 ---
-Our recommendation list contains email clients that support both [OpenPGP](encryption.md#openpgp) and strong authentication such as [Open Authorization (OAuth)](https://en.wikipedia.org/wiki/OAuth). OAuth allows you to use [Multi-Factor Authentication](basics/multi-factor-authentication.md) and prevent account theft.
+The list of recommendations here contains **email clients** that support both [OpenPGP](encryption.md#openpgp) and strong authentication such as [Open Authorization (OAuth)](https://en.wikipedia.org/wiki/OAuth). OAuth allows you to use [Multi-Factor Authentication](basics/multi-factor-authentication.md) and prevent account theft.
```
Comment (jonaharagon) [inline docs/email-clients.md:7]: Feels wordy

```suggestion
The **email clients** we recommend support both [OpenPGP](encryption.md#openpgp) and strong authentication such as [Open Authorization (OAuth)](https://en.wikipedia.org/wiki/OAuth). OAuth allows you to use [Multi-Factor Authentication](basics/multi-factor-authentication.md) and prevent account theft.
```

File: docs/encryption.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ icon: material/file-lock
 description: Encryption of data is the only way to control who can access it. These tools allow you to encrypt your emails and any other files.
 cover: encryption.webp
 ---
-Encryption of data is the only way to control who can access it. If you are currently not using encryption software for your hard disk, emails or files, you should pick an option here.
+**Encryption of data** is the only way to control who can access it. If you are currently not using encryption software for your hard disk, emails or files, you should pick an option here.
```
Comment (jonaharagon) [inline docs/encryption.md:8]:
```suggestion
**Encryption** is the only secure way to control who can access your data. If you are currently not using encryption software for your hard disk, emails, or files, you should pick an option here.
```

File: docs/news-aggregators.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: These news aggregator clients let you keep up with your favorite bl
 cover: news-aggregators.webp
 ---
 
-A [news aggregator](https://en.wikipedia.org/wiki/News_aggregator) is a way to keep up with your favorite blogs and news sites.
+A [**news aggregator**](https://en.wikipedia.org/wiki/News_aggregator) is a way to keep up with your favorite blogs and news sites.
```
Comment (jonaharagon) [inline docs/news-aggregators.md:8]:
```suggestion
A **news aggregator** is software which aggregates digital content from online newspapers, blogs, podcasts, and other resources to one location for easy viewing. Using one can be a great way to keep up with your favorite content.
```

File: docs/vpn.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ global:
 ---
 <!-- markdownlint-disable MD024 -->
 
-If you're looking for additional **privacy** from your ISP, on a public Wi-Fi network, or while torrenting files, a VPN may be the solution for you.
+If you're looking for additional **privacy** from your ISP, on a public Wi-Fi network, or while torrenting files, a **VPN** may be the solution for you.
```
Comment (jonaharagon) [inline docs/vpn.md:12]:
```suggestion
If you're looking for additional *privacy* from your ISP, on a public Wi-Fi network, or while torrenting files, a **VPN** may be the solution for you.
```

**Review by jonaharagon — COMMENTED** [review]
Awesome, thanks for doing this!

**Review by jonaharagon — APPROVED** [review]
LGTM! :)

**Review by dngray — APPROVED** [review]

**Pull request merged**